Big Integers/BigSorting.cpp

```diff
@@ -0,0 +1,78 @@
+/*
+* Title: Big Sorting
+* Description: Given a list of numbers, which would even include numbers of some 20 digit numbers or so. And your task is to sort them.
+* Author: Vanamala Srikanth
+* GitHub: https://github.com/srikanth7785
+* Twitter: https://twitter.com/srikanth7785
+*/
+
+#include<iostream>
+#include<vector>
+#include<algorithm> //for sort()
+using namespace std;
+
+bool customSorting(string a, string b){
+    //below, first two conditions, will check if one of the strings are having a `-ve`..
+    // if only first string is `-ve`
+    if(a[0] == '-' and b[0] != '-')
+        return true;
+    // if only second string is `-ve`
+    if(a[0] != '-' and b[0] == '-')
+        return false;
+    //if both are negative
+    if(a[0] == '-' and b[0] == '-'){
+        if(a.length() == b.length())
+            return a > b;
+        return a.length() > b.length();
+    }
+    // below conditions will check only when both are `+ve`
+    if(a.length() == b.length()) // if both the strings are of same length
+        return a < b; // check if first string is less than the second string
+    return a.length() < b.length(); // if both strings are of different length, then the one with smallest length will be the smallest number
+}
+
+int main(){
+    int n;
+    cin >> n;
+    vector<string> numbers;//an array can also be used instead
+    
+    for(int i = 0;i < n; i++)
+    {string temp; cin >> temp; numbers.push_back(temp);}
+    
+    sort(numbers.begin(),numbers.end(),customSorting); // this is an inbuilt method available in `algorithm`. it will call `customSorting()` for each comparision
+    
+    for(string num : numbers) //finally, print the output, which is sorted.
+    cout << num << endl;
+}
+/*
+-------------------------
+Sample Input:                                                                                   Sample Output:
```

Comment:
Please move the sample output to _after_ the input, not side-by-side. It's more difficult to copy it for testing. Use the same coding style as previously used in other programs.

Reply:
I kept them side by side so that it will be easy to understand & compare input and output.

May be, it's a bit tough to copy for testing..but, still, that's only once right..or you can just give an input on your own, right..!

Anyway, I'll change it if you want me to..

Will be waiting for your reply..🤞⏳

Reply:
Please change it. It's also hard to review since I need a wide window to view this. The program width shouldn't exceed 80 characters as an industry standard, though I'm not following that strictly.

Reply:
fine..i'll do that..now..!